control/veda_deliver.py

```diff
@@ -128,6 +129,11 @@ def run(self):
         u1.encode_size = self.video_proto.filesize
         u1.save()
 
+        # TODO: Warning! this shall be removed once 3rd party credentials
```

Comment:
I think, we should create a follow-up ticket that tracks this and when 3rd party Automated Video Transcription phases are rolled out, we would need to clean this again.

@irfanuddinahmad FYI

Reply:
we have it: https://openedx.atlassian.net/browse/EDUCATOR-1321